docs/weightonlyquant.md

```diff
@@ -115,3 +115,24 @@ gen_ids = woq_model.generate(input_ids, max_new_tokens=32, **generate_kwargs)
 gen_text = tokenizer.batch_decode(gen_ids, skip_special_tokens=True)
 print(gen_text)
 ```
+
+You can also save and load your quantized low bit model by the below code.
+
+```python
+from intel_extension_for_transformers.transformers import AutoModelForCausalLM
+
+model_path = "meta-llama/Llama-2-7b-chat-hf" # your_pytorch_model_path_or_HF_model_name
+saved_dir = "4_bit_llama2" # your_saved_model_dir
+# quant
+model = AutoModelForCausalLM.from_pretrained(model_path, load_in_4bit=True, use_llm_runtime=False)
+# save quant model
+model.save_pretrained(saved_dir)
+# load quant model
+loaded_model = AutoModelForCausalLM.from_pretrained(saved_dir)
+```
+| Inference Framework |   load GPT-Q  |    load fp32 and quantized model    |

```

Comment:
load gptq model    load the saved low-precision model from save_pretrained

Reply:
refined it in [f3b58cb](https://github.com/intel/intel-extension-for-transformers/pull/1071/commits/f3b58cb814c891a2b4f33801d303f936afa9ed0d)